kv/mdbx/kv_mdbx.go

```diff
@@ -466,6 +468,9 @@ func (db *MdbxKV) BeginRw(_ context.Context) (txn kv.RwTx, err error) {
 	}
 	runtime.LockOSThread()
 	defer func() {
+		if db.closed.Load() {
```

Comment:
We checked it in the beginning of func, adding one more check will not eliminate race